src/styles/App.css

```diff
@@ -61,13 +70,12 @@ table, th, tr, td {
   src: url("../fonts/smash_font.ttf") format("truetype"), url("../fonts/smash_font-webfont.woff") format("woff"), url("../fonts/smash_font-webfont.woff2") format("woff2");
 }
 body {
-  background: #929292;
-  background: -webkit-linear-gradient(to right, #494949, #1f1f1f);
-  background: linear-gradient(to right, #a3a3a3, #505050);
-}
-
-h1 {
-  color: blue;
+  background-image: url("../images/night_battlefield.jpg");
```

Comment:
Love the background overlay. Lets discuss if we want the same overlay on the other pages or change it up?